### PR TITLE
Update button for new build tools

### PR DESCRIPTION
**Pull request by EvanLovely**
I updated the button! 🎉

### CODE REVIEW

File: packages/components/bolt-button/package.json [hunk 1]
```diff
@@ -1,18 +1,8 @@
 {
   "name": "@bolt/components-button",
-  "patternName": "button",
-  "patternType": "components",
-  "patternSubType": "buttons",
```
Comment (remydenton) [inline packages/components/bolt-button/package.json]: @sghoweri @EvanLovely Are we removing the above lines as well?  I'm not sure what they're meant to do...

**Comment by remydenton** [comment]
I don't see step 5 here from the description in #309:

> 5. Add component to Pattern Lab - so if it was the @bolt/components-button package and it's version was 0.5.1, in the file apps/pattern-lab--workshop/package.json add it as a dependency

Was that unintentionally omitted?  Unnecessary?

File: packages/components/bolt-button/package.json (hunk 1 above)
Comment (EvanLovely) [reply packages/components/bolt-button/package.json]: @remydenton We are removing them; they were not used in any way and if we need them, it's easier and more reliable to programmatically figure them out.

**Comment by EvanLovely** [comment]
@remydenton Good catch! Got a PR up for it: https://github.com/bolt-design-system/bolt/pull/315

**Pull request merged**